packages/wasm-utils/src/utxo.rs

```diff
@@ -0,0 +1,144 @@
+use ark_bn254::Fr as Bn254Fr;
+use ark_ff::{BigInteger, PrimeField};
+use arkworks_setups::utxo::Utxo;
+use js_sys::{JsString, Uint8Array};
+use wasm_bindgen::prelude::*;
+
+pub enum JsUtxoInner {
+	Bn254(Utxo<Bn254Fr>),
+}
+
+#[wasm_bindgen]
+pub struct JsUtxo {
+	inner: JsUtxoInner,
+}
+
+impl JsUtxo {
+	/// Create `JsUtxo` from a Utxo with bn254 Fr
+	pub fn new_from_bn254_utxo(utxo: Utxo<Bn254Fr>) -> Self {
+		Self {
+			inner: JsUtxoInner::Bn254(utxo),
+		}
+	}
+
+	/// Create `JsUtxo` from a Utxo with bls381 Fr
+	pub fn new_from_bls381_utxo(_utxo: Utxo<Bn254Fr>) -> Self {
+		unimplemented!()
+	}
```

Comment:
Can you put the reason why it is not implemented for now? as in a message in the `unimplemented!("....")`

Reply:
Agreed, we can likely remove it.